plugins/destination/clickhouse/client/client_test.go

```diff
@@ -9,6 +9,14 @@ import (
 	"github.com/cloudquery/plugin-sdk/plugins/destination"
 )
 
+var migrateStrategy = destination.MigrateStrategy{
+	AddColumn:           destination.DataLossNone,
+	AddColumnNotNull:    destination.DataLossTable,
+	RemoveColumn:        destination.DataLossNone,
```

Comment:
It's literally a column loss (not in a sense that we drop tada, but in a sense that no data will be filled in after the migration to new source schema).
```suggestion
	RemoveColumn:        destination.DataLossColumn,
```

Reply:
However, maybe we need to clarify what we expect to be represented. Is it the immediate impact?